src/library/trackset/baseplaylistfeature.cpp

```diff
@@ -695,6 +695,19 @@ void BasePlaylistFeature::bindLibraryWidget(WLibrary* pLibraryWidget,
             &BasePlaylistFeature::htmlLinkClicked);
     m_pLibraryWidget = QPointer(pLibraryWidget);
     m_pLibraryWidget->registerView(m_rootViewName, pEdit);
+
+    // Update shortcuts displayed in the context menu
+    if (pKeyboard->getKeyboardConfig()->exists(ConfigKey("[Library]", "EditItem"))) {
```

Comment:
This doers not work, because we can have the situation that there is no "Edit item"  configured. 

You need either to show the correct active short cuts or nothing. Showing a not working short cut is a bug IMHO